### PR TITLE
Feature/goal add

### LINKED ISSUE #124: Goal: Add Goal tab post
A text field where user can create goal by just entering the title.

### PR DESCRIPTION
**Pull request by pallabez**
Resolves #124

### CODE REVIEW

File: interfaces/PostGoal.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface PostGoal {
```
Comment (shreya-mishra) [inline interfaces/PostGoal.ts:1]: didn't we require time duration(in which they complete the task)  field ?

**Review by shreya-mishra — APPROVED** [review]

File: interfaces/PostGoal.ts (hunk 1 above)
Comment (shreya-mishra) [reply interfaces/PostGoal.ts:1]: you have also integrated the API as well, will you please add a description as your ticket doesn't have anything regarding the api integration?

Comment (pallabez) [reply interfaces/PostGoal.ts:1]: There is `Goal` & `UserGoal`.

The UserGoal have things like, completion duration, status, derferUntil etc

File: constants/api.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export const API = {
   RDS_BASE_URL: 'https://api.realdevsquad.com',
-  GOAL_BASE_URL: 'https://website-goals-production-5d0a.up.railway.app'
+  GOAL_BASE_URL: 'https://backend-goals-production.up.railway.app'
```
[the remark this answers is not in the document]
Comment (shreya-mishra) [reply constants/api.js:3]: sure !

**Pull request merged**